modules/viewing-http-request-metrics-for-a-deployed-model.adoc

```diff
@@ -0,0 +1,43 @@
+:_module-type: PROCEDURE
+
+[id="viewing-http-request-metrics-for-a-deployed-model_{context}"]
+= Viewing HTTP request metrics for a deployed model
+
+[role='_abstract']
+
+You can view a graph that illustrates the HTTP requests that have failed or succeeded for a specific model.
+
+.Prerequisites
+* You have installed {productname-long}.
+
+* On the OpenShift cluster where {productname-short} is installed, you have enabled user workload monitoring as described in link:https://docs.openshift.com/container-platform/{ocp-latest-version}/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
+
+* You have logged in to {productname-short}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have deployed a model in a data science project.
+
+.Procedure 
+
+. From the {productname-short} dashboard navigation menu, select *Model Serving*.
+
+. On the *Model Serving* page, select the model that you are interested in.
+
+. On the *Endpoint performance* tab, optionally set the following options:
```

Comment:
Do we want to format this optional step like we do elsewhere in the RHOAI docs? That is:

```
. Optional: On the *Endpoint performance* tab, set the following options:
```